Examples/minimum_user_for_s2e/src/src_user/IfWrapper/Sils/sils_sci_if.cpp

```diff
@@ -0,0 +1,180 @@
+#pragma section REPRO
+/**
+ * @file
+ * @brief   sils_sci_if
+ * @details SCI COMでやりとりするIF
+ */
+
+#include "sils_sci_if.hpp"
+
+#ifdef WIN32
```

Comment:
[IMO]
```suggestion
#ifdef _WIN32
```

のほうがいい？

@sksat だよね？

Reply:
ですね．

まあ32と言いつつ64bitでも定義されるやつですが，ここではWindows(MSVCか)の判別をしたいだけですしそこは問題なし．

Reply:
直しましたー